<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SidebarWidget</class>
 <widget class="QDockWidget" name="SidebarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>835</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">Sidebar</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>319</width>
         <height>882</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string> Function:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fcnNameEdit">
              <property name="inputMask">
               <string notr="true"/>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QToolButton" name="showInfoButton_2">
              <property name="text">
               <string>...</string>
              </property>
              <property name="iconSize">
               <size>
                <width>8</width>
                <height>8</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
              <property name="arrowType">
               <enum>Qt::DownArrow</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="infoTitleLabel_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Information&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QToolButton" name="radarToolButton">
                <property name="toolTip">
                 <string>Radar graph</string>
                </property>
                <property name="styleSheet">
                 <string notr="true">QToolButton { /* all types of tool button */
     border: 5px solid #333;
     border-left: 10px solid #333;
	 border-right: 10px solid #333;
     border-radius: 0px;
     background-color: #333;
 }

QToolButton:hover {
     border: 5px solid #444;
     border-left: 10px solid #444;
	 border-right: 10px solid #444;
     border-radius: 0px;
     background-color: #444;
 }

QToolButton:checked {
     border: 5px solid #2180a9;
     border-left: 10px solid #2180a9;
	 border-right: 10px solid #2180a9;
     border-radius: 0px;
     background-color: #2180a9;
}

QToolTip {
	background-color: #444;
	border: 3px solid #444;
	color: rgb(232, 232, 232);
}</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../resources.qrc">
                  <normaloff>:/img/icons/radar_light.svg</normaloff>
                  <normalon>:/img/icons/radar_white.svg</normalon>
                  <activeon>:/img/icons/radar_white.svg</activeon>:/img/icons/radar_light.svg</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>8</width>
                  <height>8</height>
                 </size>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup_2</string>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="polarToolButton">
                <property name="toolTip">
                 <string notr="true">Polar graph</string>
                </property>
                <property name="styleSheet">
                 <string notr="true">QToolButton { /* all types of tool button */
     border: 5px solid #333;
     border-left: 10px solid #333;
	 border-right: 10px solid #333;
     border-radius: 0px;
     background-color: #333;
 }

QToolButton:hover {
     border: 5px solid #444;
     border-left: 10px solid #444;
	 border-right: 10px solid #444;
     border-radius: 0px;
     background-color: #444;
 }

QToolButton:checked {
     border: 5px solid #2180a9;
     border-left: 10px solid #2180a9;
	 border-right: 10px solid #2180a9;
     border-radius: 0px;
     background-color: #2180a9;
}

QToolTip {
	background-color: #444;
	border: 3px solid #444;
	color: rgb(232, 232, 232);
}</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../resources.qrc">
                  <normaloff>:/img/icons/polar_light.svg</normaloff>
                  <normalon>:/img/icons/polar_white.svg</normalon>
                  <activeon>:/img/icons/polar_white.svg</activeon>:/img/icons/polar_light.svg</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>8</width>
                  <height>8</height>
                 </size>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup_2</string>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTabWidget" name="fcnGraphTabWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>250</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>250</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Tab 1</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWebEngineView" name="fcnWebView">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>245</width>
                  <height>245</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>250</height>
                 </size>
                </property>
                <property name="url">
                 <url>
                  <string>qrc:/html/fcn_graph.html</string>
                 </url>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Tab 2</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWebEngineView" name="radarGraphWebView">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>245</width>
                  <height>245</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>250</height>
                 </size>
                </property>
                <property name="url">
                 <url>
                  <string>qrc:/html/fcn_radar.html</string>
                 </url>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="offsetToolButton">
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>8</width>
                <height>8</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
              <property name="arrowType">
               <enum>Qt::DownArrow</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Offset info:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeWidget" name="offsetTreeWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>175</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="indentation">
             <number>10</number>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string>Info</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Value</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QToolButton" name="opcodeDescButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="iconSize">
               <size>
                <width>8</width>
                <height>8</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
              <property name="arrowType">
               <enum>Qt::DownArrow</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Opcode description:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="opcodeDescText">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="undoRedoEnabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="xrefToToolButton_2">
              <property name="text">
               <string>...</string>
              </property>
              <property name="iconSize">
               <size>
                <width>8</width>
                <height>8</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
              <property name="arrowType">
               <enum>Qt::DownArrow</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="xrefToLabel_2">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>X-Refs to current address:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeWidget" name="xrefToTreeWidget_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QTreeWidget::item
{
    padding-left:10px;
    padding-top: 1px;
    padding-bottom: 1px;
    border-left: 10px;
}

QTreeWidget::item:hover
{
    background: rgb(242, 246, 248);
    color: black;
}

QTreeWidget::item:selected
{
    background: gray;
    color: white;
}

QToolTip {
	background-color: #444;
	border: 3px solid #444;
	color: rgb(232, 232, 232);
	font: 12pt &quot;Monaco&quot;;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="indentation">
             <number>0</number>
            </property>
            <column>
             <property name="text">
              <string>Address</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Instruction</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="xrefFromToolButton_2">
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>8</width>
                <height>8</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
              <property name="arrowType">
               <enum>Qt::DownArrow</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="xrefFromLabel_2">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>X-Refs from current address:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTreeWidget" name="xreFromTreeWidget_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QTreeWidget::item
{
    padding-left:10px;
    padding-top: 1px;
    padding-bottom: 1px;
    border-left: 10px;
}

QTreeWidget::item:hover
{
    background: rgb(242, 246, 248);
    color: black;
}

QTreeWidget::item:selected
{
    background: gray;
    color: white;
}

QToolTip {
	background-color: #444;
	border: 3px solid #444;
	color: rgb(232, 232, 232);
	font: 12pt &quot;Monaco&quot;;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="indentation">
             <number>0</number>
            </property>
            <column>
             <property name="text">
              <string>Address</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Instruction</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_3"/>
 </buttongroups>
</ui>
